### DISCUSSION TITLE
Initial tasks and long-term roles

### DISCUSSION THREAD

**Discussion by Falesz** [Buduf/PTI-Ear-Trainer#4] [General]
Hey all,
It's high time we started work on this project.

Once we get settled into the repo and get our first commits done, we should figure out our long-term roles for the project.
During our face-to-face meetings Márton and Dávid volunteered to develop the View layer of the program, which leaves me and André working on the Model and Persistence layers. This suits us, as far as I'm concerned. Please discuss any comments or objections regarding this matter here.

**Comment by Buduf** [comment]
These roles shouldn't be strict in my opinion. What I'm saying is that we can code in another layer if we want to help each other or one layer becomes nearly complete.
Otherwise you are right.

**Reply by Falesz** [reply]
Naturally each of us may help out in different areas of the project. What I meant was specific features should largely be done by a single person to avoid conflicts, thus we will need to utilize branching for specific features and I thought it was convenient to assign responsibilites to feature branches based on who likes to work on which layer.
